tutorials/desktop/decoding-log-files/decoding-log-files.md

```diff
@@ -0,0 +1,177 @@
+
+#### X11 server logs
+
+Location: `/var/log/Xorg.0.log`
+
+The X11 server creates a seperate log file for each of your displays. Display numbers start at zero, so your first display (display 0) will log to `Xorg.0.log`. The next display (display 1) would log to `Xorg.1.log`, and so on.
+
+
+### Non-human-readable logs
+
+Not all log files are designed to be read by humans. Some were made to be parsed by applications. Below are some of examples.
+
+#### Login failures log
+
+Location: `/var/log/faillog`
+
+Contains info about login failures. You can view it with the `faillog` command.
+
+#### Last logins log
+
+Location: `/var/log/lastlog`
+
+Contains info about last logins. You can view it with the `lastlog` command.
+
+#### Login records log
+
+Location: `/var/log/wtmp`
+
+Contains login info used by other utilities to find out who's logged in. To view currently logged in users, use the `who` command.
+
+positive
+: **This is not a complete list!**
+You can search the web for more locations relevant to what you're trying to debug. There is also a slightly more complete list [here](https://help.ubuntu.com/community/LinuxLogFiles#System_Logs).
+
+## Viewing logs using GNOME System Log Viewer
+Duration: 3:00
+
+The GNOME System Log Viewer provides a simple GUI for viewing and monitoring log files. It comes pre-installed with Ubuntu under the name `System Log`.
+
+### System Log Viewer Interface
+
+![GNOME System Log Viewer Interface](images/log-viewer-1.png)
+
+The log viewer has a simple interface. The sidebar on the left shows a list  of open log files, with the actual contents of the currently shown file displayed on the right.
+
+The log viewer not only displays but also monitors log files for changes. The bold text (as seen in the screenshot above) indicates new lines that have been logged after opening the file. When a log that is not currently selected is updated, it's name in the file list will turn bold (as shown by `auth.log` in the screenshot above).
+
+Clicking on the cog at the top right of the window will open a menu allowing you to change some display settings, as well as open and close log files.
+
+There is also a magnifying glass icon to the right of the cog that allows you to search within the currently selected log file.
+
+### More information
+
+If you wish to learn more about the GNOME System Log Viewer, you may visit the [official documentation](https://help.gnome.org/users/gnome-system-log/).
+
+## Viewing and monitoring logs in the command line
+Duration: 5:00
+
+It is also important to know how to view logs in the command line. This is especially useful when you're remotely connected to a server and don't have a GUI.
+
+There are some commands that will be useful to us when working with log files in the command line.
```

Comment:
Maybe a simpler sentence: "The following commands will be useful when working with log files."